app/views/publications/show.html.erb

```diff
@@ -11,6 +11,13 @@
                 else
                   asset_path('original/missing2.png')
                 end %>
+
+<% status = if @publication.status == "Complete" && @publication.content_date.present?
```

Comment:
let's compare with Content::COMPLETE . (defined here: https://github.com/Vizzuality/grid-arendal/blob/master/backend/app/models/content.rb#L27 )

It's basically the same, but if we mistype at least it throughs an error! =)